docs/proposals/06-immutable-etcd-backups.md

```diff
@@ -0,0 +1,315 @@
+---
+title: Immutable ETCD Backups
```

Comment:
Yes. I've already started doing so. I've been a proponent of this since the start.

For future reference, could you link the resource where you read these guidelines for me and the other maintainers as well? Thanks.